### ISSUE TITLE
MDL39 Failing unit test on Privacy provider

### ISSUE THREAD

**Issue by ScottVerbeek** [donhinkelman/moodle-block_sharing_cart#85]
Hello the plugin raises a unit test failure on the Privacy Provider test see log below.

```
There was 1 failure:

1) provider_testcase::test_table_coverage
The following tables with user fields must be covered with metadata providers: 
  - block_sharing_cart_plugins (userid)

/var/www/site/privacy/tests/provider_test.php:327
/var/www/site/lib/phpunit/classes/advanced_testcase.php:80

To re-run:
 vendor/bin/phpunit "provider_testcase" privacy/tests/provider_test.php
```

Cheers, Scott.

**Comment by golenkovm** [comment]
Hey @ScottVerbeek 
It seems the issue can be closed as fixed.
Cheers

**Comment by donhinkelman** [comment]
Thank you golenkovm for merging the fix to this issue #85.
